askup/static/assets/application.js

```diff
@@ -205,6 +205,13 @@ function on_answer_success(data) {
         });
         $('.show-on-answered').slideDown();
         $('.hide-on-answered').slideUp();
+        return;
+    }
+
+    if ('redirect_url' in data) {
+        var callback = function() {window.location.href = data.redirect_url;};
+        var notification = data.notification || 'Error on question answer, redirecting...';
```

Comment:
This is a very strange notification I would change it to some meaningful one.

Reply:
done